include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -728,6 +728,7 @@ struct Status : detail::Base<OrtStatus> {
   using Base = detail::Base<OrtStatus>;
   using Base::Base;
 
+  Status() = default;
```

Comment:
it might be worth mentioning that `Status()` is the same as `Status(nullptr)`. this is unlike other C++ API types.

also, it might be worth mentioning that such a Status represents "no error".